stock_available_to_promise_release/models/stock_move.py

```diff
@@ -387,9 +392,11 @@ def _release_split(self, remaining_qty):
         # `stock.move._search_picking_for_assignation`.
         if not self.picking_id.printed:
             self.picking_id.printed = True
-        new_move = self.browse(self._split(remaining_qty))
+
+        new_move = self.create(self._split(remaining_qty))
         # Picking assignment is needed here because `_split` copies the move
         # thus the `_should_be_assigned` condition is not satisfied
         # and the move is not assigned.
         new_move._assign_picking()
+
```

Comment:
To get some tests green, you need to call manually `_action_confirm(merge=False)` here as `_split` doesn't do it as in [13.0](https://github.com/odoo/odoo/blob/773047e9e0e73607fabef427adaa81a3a3ce657a/addons/stock/models/stock_move.py#L1522)
```suggestion
        new_move_vals = self._split(remaining_qty)
        new_move = self.create(new_move_vals)
        new_move._action_confirm(merge=False)
        # Picking assignment is needed here because `_split` copies the move
        # thus the `_should_be_assigned` condition is not satisfied
        # and the move is not assigned.
        new_move._assign_picking()
```